app/views/report/_pending.haml

```diff
@@ -0,0 +1,48 @@
+- @unreviewed_reports.each do |report|
+  - if report.item
+    .panel.panel-default
+      - username = report.user.username
+      .panel-heading
+        .reporter.pull-right
+          != t("report.reported_label", person: link_to(username, user_profile_path(username)))
+        .author
+          %span.reason-label
+            != "#{t('report.author')}:"
+          %span
+            = report.author.name
+        .created-at
+          %span.reason-label
+            != "#{t('report.created_at')}:"
+          %span
+            = I18n.l(report.created_at, format: :short)
+        .reason
+          %span.reason-label
+            = t("report.reason_label")
+          %span
+            = report.text
```

Comment:
I think this can be very confusing, what is about what. We have "Created at" which is about the report, and then we have "Author" which is the author of the reported item. I don't really know what's the best solution for this, maybe changing the text to "Reported Author/Person" makes it clearer, or maybe having two columns, one for the stuff that is about the report, and one that is about the reported item? But really, I'm mostly throwing ideas here and don't know what is best.

Also, as this information here is also interesting even if the item was deleted (for example by the author of the reported item), I would restructure the code there to always show the same things. The only difference is that if the item was deleted you can't show the text/link of the reported item anymore.

Reply:
`report.author `is now prefixed with item_author. I also changed the localized text. But you are right, we have the report author, message and creation date as well as the items (post, comments) attributes of the same meaning.

Reply:
Ah, I now see, there is already a `reported_author` in the model. I use this.